File: Benchmarks/enterprise/python/flask/testcode/benchmark_test_05611.py
# SPDX-License-Identifier: Apache-2.0
from flask import request, jsonify


def BenchmarkTest05611():
    xml_value = request.get_data(as_text=True)
    data = ' '.join(str(xml_value).split())
    data = bytearray(int(data) if str(data).isdigit() else 0)
    return jsonify({"result": "success"})

File: Benchmarks/enterprise/java/spring/testcode/BenchmarkTest05611.java
// SPDX-License-Identifier: Apache-2.0
import jakarta.servlet.http.HttpServletRequest;
import jakarta.servlet.http.HttpServletResponse;
import org.springframework.web.bind.annotation.*;

@RestController
public class BenchmarkTest05611 {

    @GetMapping("/BenchmarkTest05611")
    public void BenchmarkTest05611(@RequestHeader("Host") String host, HttpServletRequest request, HttpServletResponse response) throws Exception {
        String hostValue = host != null ? host : "";
        String data = "" + hostValue;
        int result = 100 / Integer.parseInt(data);
        response.setContentType("application/json");
        response.getWriter().print("{\"id\":0}");
    }
}
